docs/rpc-compatibility.md

```diff
@@ -0,0 +1,295 @@
+# RPC Updates to the bcoin
+
+## Ports
+Bitcoin Core does not separate wallet and node:
+
+| Network | Bcoin Node RPC | Bcoin Wallet RPC | Bitcoin Core (Wallet & Node)|
+| ---     | ---            | ---              | ---         |
+| Main    | 8332           | 8334             | 8332        |
+| Testnet | 18332          | 18334            | 18332       |
+| Regtest | 48332          | 48334            | 18443       |
+
+# List of RPC Calls in Bitcoin Core vs Bcoin
+This is a list of commands existing in core and bcoin.  
+This list currently does not include compatibility information.
+
+Note: zmq is separate from bcoin, see [bzmq](https://github.com/bcoin-org/bzmq/).
```

Comment:
I think if you make this the title for the whole doc (see above comment), you don't need these four lines.
(You DO include compatibility, right? or what do you mean here)

Reply:
yeah, this was before I included compatibility.